core/mempool/src/lib.rs

```diff
@@ -15,8 +15,13 @@ use storage::{GenericStorage, ShardChainStorage, Trie, TrieUpdate};
 const POISONED_LOCK_ERR: &str = "The lock was poisoned.";
 
 pub mod pool_task;
+pub mod tx_gossip;
 
 use crate::pool_task::MemPoolControl;
+use primitives::signature::SecretKey;
```

Comment:
secret keys shouldn't be sent around. We have {Block/Transaction}Signer interface instead.

See #653 for the propagation of that.